README.md

```diff
@@ -30,9 +30,10 @@ users = {
 
 @auth.verify_password
 def verify_password(username, password):
-    if username in users and \
-            check_password_hash(users.get(username), password):
-        return username
+    # Check a password hash, even if user doesn't exist to prevent timing attacks
+    if check_password_hash(users.get(username, 'not-the-password'), password):
```

Comment:
Does this achieve anything? The `check_password_hash()` function takes a hash as first argument, and you are passing something that is not that. How can the function go through the motions of checking a real hash if you don't pass one?